views/js/qtiCreator/editor/mathInput/tpl/mathInput.tpl

```diff
@@ -1,6 +1,6 @@
 <div class="math-input">
```

Comment:
I have a weird issue (on TAE): 
mousedown and start selecting (from right to left), then mouseup outside of dialog window --> Unsaved changes dialog appears.
Don't know if it's related ot not.

https://user-images.githubusercontent.com/38751932/178195789-50eca762-c4d4-4b48-96dc-bb448f594d1b.mp4

Reply:
Ir reproduces also on existing version, so it's not related to this PR. Let me know if you don't plan on fixing it here, and I'll approve.